lib/modules/carto.rb

```diff
@@ -30,6 +44,16 @@ def total_area_query
     query
   end
 
+  def total_area_by_country_query
+    query = "SELECT SUM(gis_area_k),iso3 FROM #{from} GROUP BY iso3 ORDER by SUM(gis_area_k) desc"
```

Comment:
Creating a variable here is not necessary, as the method will always return the output of the last line of code inside it. So the string only is sufficient.

Reply:
Thanks!